WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/BloggingPromptsAttribution.swift

```diff
@@ -0,0 +1,44 @@
+enum BloggingPromptsAttribution: String {
+    case dayone
+
+    var attributedText: NSAttributedString {
+        let baseText = String(format: Strings.fromTextFormat, source)
+        let attributedText = NSMutableAttributedString(string: baseText, attributes: Constants.baseAttributes)
+        guard let range = baseText.range(of: source) else {
+            return attributedText
+        }
+        attributedText.addAttributes(Constants.sourceAttributes, range: NSRange(range, in: baseText))
+
+        return attributedText
+    }
+
+    var source: String {
+        switch self {
+        case .dayone: return Strings.dayOne
+        }
+    }
+
+    var iconImage: UIImage? {
+        switch self {
+        case .dayone: return Constants.dayOneIcon
+        }
+    }
+
+    private struct Strings {
+        static let fromTextFormat = NSLocalizedString("From %@", comment: "From string format for blogging prompts attribution")
```

Comment:
Some comments about this string:
- In string formats, we always use positional placeholders. i.e. this should be `%1$@` instead of `%@`. (Ref pbMoDN-1Df-p2).
- We explain what each placeholder is for in the `comment`.
- The `comment` is a bit confusing due to the usage of `From` and its context.

Maybe it can be something like:
`NSLocalizedString("From %1$@", comment: "Format for blogging prompts attribution. %1$@ is the attribution source.")`

Reply:
Thanks for the catch and the info! I updated the PR with your recommendation: https://github.com/wordpress-mobile/WordPress-iOS/pull/18574/commits/760e5761320ee44a86a12f1245cb10d63a57d663